mission/battery_monitor/src/battery_monitor.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+#python imports
+import subprocess
+import os
+
+#ros imports
+import rospy
+from std_msgs.msg import Int32
+
+class BatteryMonitor():
+    def __init__(self):
+        
+        rospy.init_node("battery_monitor")
+
+        # Settings
+        self.path_to_voltage_meter = "/sys/bus/i2c/drivers/ina3221x/1-0040/iio:device0/in_voltage0_input"
+        self.frequency = 10.0       # How often the battery level is checked and published
+
+        # Thresholds
+        self.critical_level = 14000 # Minimum allowed value in millivolts
+        self.warning_level = 15000  # Warning threshold in millivolts
+        
```

Comment:
It would be more future proof if these were provided through rosparams from the drone.yaml files in auv_config

Reply:
This also applies to 
```python        
self.path_to_voltage_meter = "/sys/bus/i2c/drivers/ina3221x/1-0040/iio:device0/in_voltage0_input"
self.frequency = 10.0
```

Reply:
Do you want me to add the parameters in one of the yaml files? In that case, which one?